templates/accounts/volunteer_profile.html

```diff
@@ -167,6 +167,26 @@ <h2 class="font-weight-bolder">Complete Your Profile</h2>
 
 
 {% block scripts %}
+    <!--  script for AJAX call -->
+    <script>
+    $(document).ready(function(){
+        $("#rollNo").focusout(function(){
+            console.log('roll');
+            $.post("./../../accounts/ajax_volun_roll/",
+            {
+                roll: document.getElementById('rollNo').value,
+                'csrfmiddlewaretoken': '{{ csrf_token }}',
+            },
+            function(data,status){
+                if (data.isExist) {
+                    console.log(data);
+                    document.getElementById('duplicate').innerHTML = "Duplicate roll encountered.";
+                }
```

Comment:
Replace this `if` block with the following:
```
if (data.isExist) {
    document.getElementById("rollNo").setCustomValidity("Duplicate roll encountered.");
}
else {
    document.getElementById("rollNo").setCustomValidity("");
}
```

Reply:
`setCustomValidity` actually triggers that invalid-feedback class division below roll_no but can not change the error message. Apart from the error message all functionalities working properly.

Reply:
Yeah, I know. You need to think of some way to solve that.